cura/resources/texts/whats_new/2.html

```diff
@@ -1,3 +1,2 @@
-<h4>Cloud printing and new materials for Method series printers</h4>
-<p><a href="https://ultimaker.com/3d-printers/method-series/?utm_source=whatsnew&utm_medium=cura&utm_campaign=57">The UltiMaker Method and Method XL printers</a> can now send prints over the cloud directly from Cura and can also use the UltiMaker Digital Factory to organize print files and monitor ongoing prints. Additionally, three new material profiles have been added for the Method XL in this release. They are ASA, SR-30, and Nylon12-CF.
-</p>
\ No newline at end of file
+<h4>Search bar added to “Add an offline printer” list  </h4>
+<p>A search bar has been added that lets you search for the print profile you need when adding a printer. Making it quicker than ever to get up and running, whether you’re using an UltiMaker printer or one of the hundreds of 3rd party printers that are natively supported within Cura.</p>
```

Comment:
Search for the print profile ? Well you search for a brand/printer, not for a profile I would say